package.json

```diff
@@ -2,8 +2,11 @@
   "name": "service",
   "version": "2.3.0",
   "description": "Service side of clearlydefined.io.",
+  "engines": {
+    "node": "18 || 24"
```

Comment:
I would just drop Node v18 at this point. No need to worry about backward compatibility when we're moving to v24.